agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/SnippetConfiguration.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.configuration;
+
+import io.opentelemetry.javaagent.bootstrap.servlet.ExperimentalSnippetHolder;
+
+// preveiw Configuration
+public class SnippetConfiguration {
+  private static String snippet =
```

Comment:
can customer modify snippet? has to be in this format? is there a doc to understand this better like what it does?

Reply:
Customer could set their own snippet via otel snippet set, the one here we provided is for the application-insights web snippet so it does not have much flexibility.

Reply:
The doc would be in a new pr soon.